src/main/java/org/galatea/starter/domain/Quote.java

```diff
@@ -0,0 +1,15 @@
+package org.galatea.starter.domain;
+
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
```

Comment:
Does it need to be ```@Data``` or should it be ```@Value```?

Reply:
We can make these objects immutable. However in the case of Entity and Entity store objects we cannot use @Value because @Value uses @FieldDefaults(makeFinal=true, level=AccessLevel.PRIVATE) which means we cannot omit any fields from our constructor. 
Instead of @Value, we can use @Getter, @ToString, @EqualsAndHashCode, @RequiredArgsConstructor